.github/workflows/cpp_package.yaml

```diff
@@ -187,6 +187,35 @@ jobs:
                   name: cpp_mcu_bin-${{ matrix.idf_target }}
                   path: ${{ runner.workspace }}/cppbuild/Slint-cpp-*
 
+    cmake_package_arm_linux:
+        env:
+            CARGO_INCREMENTAL: false
+        strategy:
+            matrix:
+                gcc_target: [aarch64-linux-gnu, arm-linux-gnueabihf]
+                include:
+                  - gcc_target: aarch64-linux-gnu
+                    rust_target: aarch64-unknown-linux-gnu
+                  - gcc_target: arm-linux-gnueabihf
+                    rust_target: armv7-unknown-linux-gnueabihf
+
+        runs-on: ubuntu-20.04
```

Comment:
how about 24.04 ?

Reply:
I could do that, sure. (Doesn't really matter so much as the build is run inside the container anyway).

Reply:
Just to delay the moment where we'll have warning about using deprecated platform